api/StorageEstimate.json

```diff
@@ -20,7 +20,7 @@
             "version_added": "51"
           },
           "ie": {
-            "version_added": null
+            "version_added": false
```

Comment:
This is from https://storage.spec.whatwg.org/, a spec that didn't exist at the time of IE11.